src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
Well, for one thing, it is going to be difficult to start choosing which sets of defaults we want to choose in one go. I rather we do it slowly, for example, if in this case, we decide that this inspection is to be turned off, we turn it off in this PR.

As for making the code unreadable, that is only if the comment is useless, which in this case it isn't. It points out something is not wanted, which in some cases the comment may not be clear on what it is doing we can explain it.

Now, why not no comments altogether so users do not have to be distracted at all. However, in this case, no matter whether a person is using an IDE or not it isn't a distraction because we are not following something commonly recommended in CSS, and we should be pointing it out.